packages/css/src/components/button/button.scss

```diff
@@ -3,47 +3,96 @@
  * Copyright (c) 2023 Gemeente Amsterdam
  */
 
-@import "../../../node_modules/@utrecht/components/button/css";
-
-@mixin reset {
-  -webkit-text-size-adjust: 100%;
-}
-
-.amsterdam-button {
+@mixin amsterdam-button {
```

Comment:
What’s the added value of using mixins? A button already has both `.amsterdam-button` and `.amsterdam-button-${variant}`, so we could use classes as well. No other components use the mixins approach.